test/slack_test.rb

```diff
@@ -128,6 +128,20 @@ def test_v2_custom_alerts
       assert_not_nil(attachment["fields"])
       assert_equal(1, attachment["fields"].length)
       assert_not_nil(attachment["mrkdwn_in"])
+      assert (not attachment["pretext"].include?('This is a test message'))
```

Comment:
So you suggestion is:

"This is a test message sent by #{email} via metrics.librato.com/alerts. No action is required."

should be replaced by

"This is a test alert notification sent by #{email]} via metrics.librato.com/alerts. No action is required."

I also like the latter better. It will also make us suffer because of the duplication ;o)